osCam/userconfig/models.py

```diff
@@ -1,24 +1,69 @@
     network = models.ForeignKey(Network, on_delete=models.DO_NOTHING, null=True, blank=True)
-    model_num = models.IntegerField(blank=True, null=True)
-    model_name = models.CharField(max_length=40,blank=True, null=True)
+    modelNum = models.IntegerField(blank=True, null=True)
+    modelName = models.CharField(max_length=40,blank=True, null=True)
+    username = models.CharField(max_length=40, default='admin')
+    password = models.CharField(max_length=40, default='admin')
+
+    def __str__(self):
+        return self.modelName
 
 class Camera(models.Model):
-    # Rasberry Pi can have Many Camera (1:Many)
     user = models.ForeignKey( User, on_delete=models.CASCADE, null=True, blank=True )
-    raspberry_pi = models.ForeignKey( RaspberryPi , on_delete=models.CASCADE, null=True, blank=True)
-    model_num = models.IntegerField(blank=True, null=True)
-    model_name = models.CharField(max_length=40,blank=True, null=True)
+    raspberryPi = models.ForeignKey( RaspberryPi , on_delete=models.CASCADE, null=True, blank=True)
+    modelNum = models.IntegerField(blank=True, null=True)
+    modelName = models.CharField(max_length=40,blank=True, null=True)
+    cameraIndex = models.IntegerField(default=0, unique=False, editable=False)
+    deviceName = models.CharField(max_length=40, default='Camera 01')
+    ipAddress = models.CharField(max_length=40, default='10.0.0.94')
+    port = models.IntegerField(default=80)
+
+    def __str__(self):
+        return self.deviceName
+
+class CameraView(models.Model):
+    user = models.ForeignKey( User, on_delete=models.CASCADE, null=True, blank=True )
+    # camera = models.ForeignKey( Camera, on_delete=models.CASCADE, null=True, blank=True)
+    showMotionBoxes = models.BooleanField(default=False)
+    showContours = models.BooleanField(default=False)
+    showText = models.BooleanField(default=False)
+    text = models.CharField(max_length=255, default='Cam Name :: Date :: Time')
+    contrast = models.IntegerField(default=0, blank=True, null=True)
+    brightness = models.IntegerField(default=0, blank=True, null=True)
+    recording = models.BooleanField(default=False, blank=True, null=True)
+    fps = models.IntegerField(default=15)
+    invert = models.BooleanField(default=False)
+    mirror = models.BooleanField(default=False)
+
+    def __str__(self):
+        return "Camera View settings"
+
+class Storage(models.Model):
+    recordToDevice = models.BooleanField(default=False)
+    recordToCloud = models.BooleanField(default=False)
+    filePath = models.CharField(max_length=255, default='/home/pi/Videos')
+    maxSpace = models.IntegerField(default=0, blank=True, null=True)
+    timeToLive = models.IntegerField(default=0, blank=True, null=True)
+    archive = models.BooleanField(default=False, blank=True, null=True)
+    lengthOfRecordings = models.IntegerField(default=0, blank=True, null=True)
+    codec = models.CharField(max_length=40, default='h264') 
```

Comment:
I think I am currently using a different default for the codec, have you validated that 'h264' works?

Reply:
have not this is all dummy stuff. it updates the model in the tables but doesnt really touch anything you're doing. can be molded to whatever works for you. for now we should set it to all our working defaults